docs/modules/ROOT/pages/tutorials/aws.adoc

```diff
@@ -0,0 +1,48 @@
+== AWS - Amazon Web Services Load Balancers
```

Comment:
This is not a tutorial. For me this is a how to but a discussion would also be fine.